portafly/src/App.tsx

```diff
@@ -1,12 +1,14 @@
+// Included in create-react-app
+// eslint-disable-next-line import/no-extraneous-dependencies
 import 'react-app-polyfill/ie11'
-import { Brand } from '@patternfly/react-core'
 import React from 'react'
-import {BrowserRouter as Router, Redirect, useHistory} from 'react-router-dom'
-import { AppLayout, SwitchWith404, LazyRoute } from 'components'
+import { Redirect, BrowserRouter as Router, useHistory } from 'react-router-dom'
+import { Brand } from '@patternfly/react-core'
+import { AppLayout, LazyRoute, SwitchWith404 } from 'components'
 import { LastLocationProvider } from 'react-router-last-location'
 import logo from 'assets/logo.svg'
 
-const Logo = <Brand src={logo} alt={'patternfly logo'}/>
+const Logo = <Brand src={logo} alt="patternfly logo" />
```

Comment:
Is there a reason to not always use single quotes?

Reply:
The reason is you would use doubles quotes in an HTML file and it's become a standard defacto to considered JSX the same way. There is no practical reason, I think.

Reply:
I'm not convinced with this argument of: use double quotes when html, use single with js. For me it's all js as we are using jsx.
But again, I can adapt to both styles

Reply:
I'm not trying to convince anyone: it's just a standard people use, just like using camelCase instead of PascalCase. It comes from this kind of situation:
```js
<input value="user's choice"/>
```
vs
```js
<input value='user\'s choice'/>
```

Reply:
We could use template strings, but maybe it's too much. OK, let's use double